pandas/io/parquet.py

```diff
@@ -0,0 +1,154 @@
+""" parquet compat """
+
+from warnings import catch_warnings
+from pandas import DataFrame, RangeIndex, Int64Index
+from pandas.compat import range
+
+
+def get_engine(engine):
+    """ return our implementation """
+
+    if engine not in ['pyarrow', 'fastparquet']:
+        raise ValueError("engine must be one of 'pyarrow', 'fastparquet'")
+
+    if engine == 'pyarrow':
+        return PyArrowImpl()
+    elif engine == 'fastparquet':
+        return FastParquetImpl()
+
+
+class PyArrowImpl(object):
+
+    def __init__(self):
+        # since pandas is a dependency of pyarrow
+        # we need to import on first use
+
+        try:
+            import pyarrow  # noqa
```

Comment:
sorted; I realized that ``pyarrow.parquet`` is an explicit import which is fine.